File: ria/java/gof/src/main/java/com/gof/pattern/factoryMethod/after/Ship.java
package com.gof.pattern.factoryMethod.after;

import com.gof.pattern.abstractFactory.after.Anchor;
import com.gof.pattern.abstractFactory.after.Wheel;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Ship {
    private String name;

    private String color;

    private String logo;

    private Anchor anchor;

    private Wheel wheel;
}

File: ria/gof/src/main/kotlin/com/gof/pattern/factory/Ship.kt
package com.gof.pattern.factory

import com.gof.pattern.abstractFactory.after.Anchor
import com.gof.pattern.abstractFactory.after.Wheel

open class Ship {
    var name: String? = null
    var color: String? = null
    var logo: String? = null
    var anchor: Anchor? = null
    var wheel: Wheel? = null
}
